orchestration/src/main/java/com/sap/ai/sdk/orchestration/UserMessage.java

```diff
@@ -13,5 +18,90 @@ public class UserMessage implements Message {
   @Nonnull String role = "user";
 
   /** The content of the message. */
-  @Nonnull String content;
+  @Nonnull
+  @Getter(onMethod_ = @Beta)
+  MessageContent content;
+
+  /**
+   * Creates a new user message from one or more strings.
+   *
+   * @param message the first message.
+   * @param additionalMessages the additional messages.
+   */
+  public UserMessage(@Nonnull final String message, @Nullable final String... additionalMessages) {
+    content = MessageContent.text(message, additionalMessages);
```

Comment:
(Minor)
Please use constructor chaining: `this(MessageContent.text(message, additionalMessages));`